README.md

```diff
@@ -53,35 +56,32 @@ There are three groups of tests:
 
 * Unit tests
   * Simple unit tests for individual functions and classes.
-  * **Framework:** Mocha
-  * **Assertions:** Chai
+  * **Framework:** [Vitest](https://vitest.dev/)
+  * **Assertions:** [Chai](https://www.chaijs.com/)/[Jest](https://jestjs.io/docs/expect)
   * **Path:** `tests/unit`
-  * **Command:** `yarn run test:unit`
-  * (To watch add `--watch`)
+  * **Command:** `yarn run test:unit` (watches by default, only re-runs changed file)
+  * (To prevent watching, use `yarn vitest run`)
```

Comment:
```suggestion
  * **Command:** `yarn run test:unit` (watches by default, only re-runs changed file:
    To prevent watching, use `yarn vitest run`)
```